lib/challenge.js

```diff
@@ -61,6 +78,17 @@ var some_pet;
 var Pet;
 // Answer Starts Here
 
+Pet = function Pet (name, address) {
+  this.name = name;
+  this.address = address;
+  this.some_pet = [];
+}
+
+Pet.prototype.specifics = function specifics () {
+  var name = some_pet.name
+  return name;
+};
```

Comment:
Not quite. The method should be `makeNoise`, and it should reference `this.noise`, like so:
```javascript
Pet.prototype.makeNoise = function() {
  console.log(this.noise);
};
```